src/NuGet.Clients/NuGet.CommandLine/Commands/RestoreCommand.cs

```diff
@@ -283,7 +283,7 @@ private async Task<IReadOnlyList<RestoreSummary>> PerformNuGetV2RestoreAsync(Pac
                 {
                     foreach (PackageReference packageReference in GetInstalledPackageReferences(configFile))
                     {
-                        if (configToProjectPath.TryGetValue(configFile, out string projectPath))
+                        if (!configToProjectPath.TryGetValue(configFile, out string projectPath))
```

Comment:
This looks like you fixed a bug?

Reply:
yep, details about where a package was coming were never truly relevant before. 

Now that we have a config that's project specific that needs looked up after restore, it is relevant.